cip.go

```diff
@@ -147,17 +169,18 @@ func main() {
 	// the manifest desires a completely blank registry. In practice this would
 	// almost never be the case, so given a fully-parsed manifest with 0 images,
 	// treat it as if -parse-only was implied and exit gracefully.
-	if len(mfest.Images) == 0 {
-		fmt.Println("No images in manifest --- nothing to do.")
-		os.Exit(0)
-	}
+	if len(*snapshotPtr) == 0 {
+		if len(mfest.Images) == 0 {
+			fmt.Println("No images in manifest --- nothing to do.")
```

Comment:
Small nit that doesn't need to be addressed in this PR. I see a lot of `fmt.Print` in this file. Do we want to use a logger?

Reply:
Hmm well we do want to always print that string regardless of the logging level. Not sure if that makes sense in a logging-level context.

Even if we should use a logger, I think that can be a separate PR (don't wanna change the hash of this commit to throw off #62 which depends on this).